OJP/OJP_JourneySupport.xsd

```diff
@@ -68,7 +94,14 @@
 					<xs:documentation>Public code of the SERVICE JOURNEY (same meaning as in NeTEx). Examples: "512", "S8" or "Circle Line" or "ICE 488".</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:group ref="ServiceGroup"/>
+			<xs:choice>
```

Comment:
I really really really need to be convinced we need a choice here.

Reply:
I thought it would be good for backwards compatibility, but you are right, not having a StopSequenceIntervalGroup results in exactly the same substructure. I remove it.